### PR TITLE
refactor: change commonjs to modulejs

### PR DESCRIPTION
**Pull request by LongTCH**

### CODE REVIEW

File: configs/env.config.js [hunk 1]
```diff
@@ -1,11 +1,16 @@
-require('dotenv').config();
+const CLIENT_URL = process.env.DEV_CLIENT_URL || 'http://localhost:3000';
+const SERVER_URL = process.env.DEV_SERVER_URL || 'http://localhost:8080';
+const DB_CONNECTION_STRING =
+	process.env.DB_CONNECTION_STRING || 'mongodb://127.0.0.1:27017';
```
Comment (ces-hoangdao) [inline configs/env.config.js:4]: Should use mongo cloud

File: configs/env.config.js [hunk 2]
```diff
@@ -1,11 +1,16 @@
-require('dotenv').config();
+const CLIENT_URL = process.env.DEV_CLIENT_URL || 'http://localhost:3000';
+const SERVER_URL = process.env.DEV_SERVER_URL || 'http://localhost:8080';
+const DB_CONNECTION_STRING =
```
Comment (ces-hoangdao) [inline configs/env.config.js:3]: file name  should config.js

File: .env.example [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+
+PORT=8080
+
+CLIENT_ID='53111155958-trfcu27tsaaicdqbud03p6pfgmkocdu9.apps.googleusercontent.com'
+
+CLIENT_SECRET='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'
+
+DEV_CLIENT_URL='http://localhost:3000'
+
+DEV_SERVER_URL='http://localhost:8080'
```
Comment (ces-hoangdao) [inline .env.example:11]: add 1 empty line here

**Review by ces-hoangdao — APPROVED** [review]

File: configs/env.config.js (hunk 1 above)
Comment (LongTCH) [reply configs/env.config.js]: yes

**Pull request merged**